CMake/ensmallen-config.cmake.in

```diff
@@ -0,0 +1,4 @@
+@PACKAGE_INIT@
+
+include(${CMAKE_CURRENT_LIST_DIR}/@targets_export_name@.cmake)
+check_required_components(ensmallen)
```

Comment:
Newline at the end?

Reply:
Thanks, handled during merge with 67a94b2. :+1: